ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml

```diff
@@ -527,6 +527,47 @@ tests:
     - ref: ipi-install-heterogeneous
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-ovn-ipsec
+- as: gcp-ipi-user-labels-tags-tp-amd-mixarch-f28-destructive
```

Comment:
I suggest to run arm arch install for non-destructive and amd arch install for destructive using the same multi-arch payload for the cross coverage. So the job name can be `gcp-ipi-labels-tags-filestore-csi-tp-amd-f28-destructive`

Reply:
An example is [here](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml#L358-L387).

Reply:
So no need to use mixarch?

Reply:
right, generally `mixarch` means the workers mixed different arch in this cluster.